### PR TITLE
hack: add controller-manager and mount-utils repo

### PR DESCRIPTION
**Pull request by nikhita**
For https://github.com/kubernetes/org/issues/1895 (PR to add staging dir at https://github.com/kubernetes/kubernetes/pull/91354)

/hold
until https://github.com/kubernetes/kubernetes/pull/91354 and https://github.com/kubernetes/test-infra/pull/17712 have merged

/assign @dims

### CODE REVIEW

**Comment by nikhita** [comment]
Note: this does not _enable_ publishing-bot for the repo. The publishing-bot is already running and once the config is updated, the bot will sync staging with the repo.

This script is manually invoked while testing using our forks and is used to sync upstream branches with the forked repos. 

The hold is just to ensure that we get one complete run of the publishing-bot to sync the controller-manager repo before bumping the image.

**Comment by nikhita** [comment]
Also added mount-utils for https://github.com/kubernetes/org/issues/1887

**Comment by nikhita** [comment]
/hold cancel

repos have been created

File: hack/fetch-all-latest-and-push.sh [hunk 1]
```diff
@@ -43,6 +43,7 @@ repos=(
     cluster-bootstrap
     code-generator
     component-base
+    controller-manager
```
Comment (sttts) [inline hack/fetch-all-latest-and-push.sh:46]: can't we extract this from the yaml config?

**Comment by dims** [comment]
Do we still need this @nikhita ?

**Comment by nikhita** [comment]
Closing, will create another PR to automatically derive this from rules.yaml

**Pull request closed without merging**